src/posix_runner_platform.c

```diff
@@ -82,10 +82,13 @@ void run_specified_test_if_child(TestSuite *suite, TestReporter *reporter){
 
 static void stop(void) {
 #ifdef INTERNAL_WITH_GCOV
```

Comment:
I think we could get rid of this `#ifdef`. We'd just have to modify the .travis.yml script to set `CGREEN_CHILD_EXIT_WITH_FLUSH` when running the tests.